### ISSUE TITLE
Unable to upload larger mp4 files using IIS

### ISSUE THREAD

**Issue by Zeegaan** [umbraco/Umbraco-CMS#11800]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

9.1.2

### Bug summary

When trying to upload a mp4 file to the media section, with a file size of 200mb, you're unable to upload the file, this will not give any visible errors of any kind, only if you open dev tools and look at the console you get this error: 
![image](https://user-images.githubusercontent.com/70372949/147638446-345e1ad8-ae57-48b5-86ea-6804cb220e81.png)
Seems the file is to large and it hits the cap, there should be no cap after the fix in https://github.com/umbraco/Umbraco-CMS/pull/11052

### Specifics

I've tried this on 9.0.0 and we actually get a visible error in the browser, not just the console, but still an error nonetheless

### Steps to reproduce
- Run umbraco using IIS, kestrel seems to work fine
- Download a sample file of size that is bigger than 28.6mb and less than ~4gb (this is the hard cap for IIS Express) this should work with any uploadable filetype, like jpeg, png, mp4 etc.
- Navigate to the media section of umbraco, (with dev tools open)
- Try and upload the file

### Expected result / actual result

The file should be uploaded unless a max file size has been set

**Comment by Zeegaan** [comment]
Update: I've changed this issue to only be about IIS, I've tried this with kestrel and everything works as intended

**Comment by Zeegaan** [comment]
Update: I've found an easy workaround if the matter is pressing:

Create a Web.config file in the root of your project and put this in:
This will configure the IIS maxAllowedContentLength manually
```
<?xml version="1.0"?>
<configuration>
  <system.webServer>
    <staticContent>
      <clientCache cacheControlCustom="private" cacheControlMode="UseMaxAge" cacheControlMaxAge="3.00:00:00" />
    </staticContent>
    <security>
      <requestFiltering>
        <!-- ~ 4GB -->
        <requestLimits maxAllowedContentLength="4294967295" />
      </requestFiltering>
    </security>
  </system.webServer>
</configuration>

```

**Comment by nikolajlauridsen** [comment]
This has been fixed in #11802 by obsoleting the `ConfigureIISServerOptions`, since this is no longer possible, for more information about increasing the limit with IIS see: https://our.umbraco.com/documentation/Reference/V9-Config/MaximumUploadSizeSettings/
